lumicks/pylake/kymo.py

```diff
@@ -577,6 +577,7 @@ def estimate_bead_edges(
         algorithm,
         *,
         channel="green",
+        slack=0,
```

Comment:
I don't think anyone is going to understand the intention of this argument from the name. Maybe something like `padding` or `extra_cropping`, `additional_cropping`?

Reply:
Fair point. Let's go with `extra_cropping`. `padding` makes me think we go towards the outside and additional is just longer.